src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ CHANGELOG
  * Deprecate constraint `ExpressionLanguageSyntax`, use `ExpressionSyntax` instead
  * Add method `__toString()` to `ConstraintViolationInterface` & `ConstraintViolationListInterface`
  * Allow creating constraints with required arguments
+ * Add the `match` option to the `Choice` constraint
```

Comment:
I had to read the description of the PR to get what this was for.
Maybe it's a naming issue? Would `negate` be better instead? I'm not sure, just trying :)

Reply:
Another possible naming is `shouldMatch`, as it's more clear to understand that a boolean is expected.

Reply:
I've spent some time thinking about this. I didn't find any reasonable alternative. The whole concept seems like an edge case anyway, so let's keep `match` (at least, this is the same name as for Regexp). Doc should be well written though :)

Reply:
And of course, if anyone has a better name before 6.2 final, we will be able to change it quickly.
cc @symfony/mergers